lib/google_scraping/dashboard/keywords.ex

```diff
@@ -103,6 +109,43 @@ defmodule GoogleScraping.Dashboard.Keywords do
     |> Repo.update!()
   end
 
+  def store_keyword_urls(keyword, urls, is_ad) do
+    keyword_urls =
```

Comment:
Can we move it to a separate function? may be `build_keyword_urls(..)`

Reply:
Fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/52/commits/b23a2026db2ae21b6a3f5afc0b8896c833430f88